src/libcollections/btree/set.rs

```diff
@@ -11,12 +11,12 @@
 // This is pretty much entirely stolen from TreeSet, since BTreeMap has an identical interface
 // to TreeMap
 
-use core::cmp::Ordering::{self, Less, Greater, Equal};
-use core::cmp::{min, max};
+use core::cmp::Ordering::{self, Equal, Greater, Less};
+use core::cmp::{max, min};
```

Comment:
I think that this should be `min, max` and `Less, Equal, Greater`. Mostly, I like alphabetization, but when the things explicitly come with an ordering, I think it makes sense to go with that ordering.